scripts/version_maker.sh

```diff
@@ -18,63 +22,126 @@ function yaml() {
     }'
 }
 
-# find files ended with chartserviceversion.yaml 
-FIND_CMD=`find ./charts -type f -name "*basetemplate.yaml"`
+# This function checks whether the kind is CSV or CE
+checker(){
+    if [[ $1 == "ChaosExperiment" ]]; then
+        echo $metadata_version
+    elif [[ $1 == "ChartServiceVersion" ]]; then
+        echo $spec_version 
+    fi
+}
 
```

Comment:
Let us change the experiment and charts to get version from same place in the spec. Say, metadata